src/lib/forms/UserSubmissionForm.svelte

```diff
@@ -0,0 +1,172 @@
+<script lang="ts">
+	import { enhance } from '$app/forms';
+	import { form_action } from '../form_action';
+	import { Turnstile } from 'svelte-turnstile';
+	import { env } from '$env/dynamic/public';
+	import InlineError from './InlineError.svelte';
+	export let form;
+	let reset: () => void | undefined;
+</script>
+
+<form
+	action="?"
+	method="post"
+	use:enhance={form_action(undefined, undefined, (data, context) => {
+		if (data.status === 200) {
+			return context?.formElement.reset();
+		}
+		if (data.error) {
+			console.info('Resetting turnstile');
+			reset();
+		}
+	})}
+>
+	<div class="input-group">
+		<label for="submission_type">Submission Type</label>
+		<div class="input">
+			<InlineError displayError={form?.fieldErrors?.['submission_type']} />
+			<select name="submission_type" id="submission_type">
+				<option value="POTLUCK">Potluck Question</option>
+				<option selected value="SPOOKY">👻🎃 Spooky Story Submission</option>
+				<option value="GUEST">Guest Suggestion</option>
+				<option value="FEEDBACK">Show Feedback</option>
+				<option value="OTHER">Other</option>
+			</select>
+		</div>
+	</div>
+	<div class="input-group">
+		<div>
+			<label for="body">What's Up?</label>
+			<p class="required">Required</p>
+		</div>
+		<div class="input">
+			<InlineError displayError={form?.fieldErrors?.['body']} />
+			<textarea
+				id="body"
+				name="body"
+				placeholder="What do you have to say? Write as much as you like!"
+				required
+				maxlength={15000}
+			></textarea>
+		</div>
+	</div>
+
+	<div class="input-group">
+		<label for="name">Name</label>
+		<div class="input">
+			<InlineError displayError={form?.fieldErrors?.['name']} />
+			<input
+				type="text"
+				id="name"
+				name="name"
+				placeholder="Name, @handle, or Alias"
+				maxlength={100}
+			/>
+		</div>
+	</div>
+
+	<div class="input-group">
+		<label for="email">Email</label>
+		<div class="input">
+			<InlineError displayError={form?.fieldErrors?.['email']} />
+			<input
+				type="email"
+				id="email"
+				name="email"
+				placeholder="Only if we need it"
+				maxlength={100}
+			/>
+		</div>
+	</div>
+	<div class="input-group turnstile">
+		<Turnstile siteKey={env.PUBLIC_TURNSTILE_SITE_KEY} bind:reset />
+	</div>
+	<button type="submit">Send</button>
+</form>
+{#if form?.error}
```

Comment:
Thoughts on moving this above the form? Had to look around to find the success message.

Reply:
yeah good call. Not 100% happy with the layout yet but its better